internal/controller/api/management.go

```diff
@@ -38,7 +38,9 @@ func (s *ManagementServer) Routes() {
 	securedSubRouter := s.router.PathPrefix("/connection").Subrouter()
 	amw := &middlewares.AuthMiddleware{Secrets: s.config.ServiceToServiceCredentials}
 	securedSubRouter.Use(amw.Authenticate)
+	//securedSubRouter.Use(handlers.CORS())
```

Comment:
Handling CORS was required to use this api locally, but shouldn't be necessary when going through 3scale so I've left this middleware commented out for now.